lib/tbot/cli/db.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cli
+
+import "context"
+
+// DBCommand contains fields for `tbot db`
+type DBCommand struct {
+	*genericExecutorHandler[DBCommand]
+
+	DestinationDir string
+	Cluster        string
+	ProxyServer    string
+
+	// LegacyProxy is the legacy --proxy flag.
+	// TODO(timothyb89): DELETE IN 17.0.0
+	// TODO(timothyb89): Or maybe remove in this PR.
+	LegacyProxyFlag string
```

Comment:
@strideynet do we just want to remove this flag now? It's ported over as is, but I figured it was worth discussing. Happy to follow up with a removal later.

Reply:
Let's leave it in for now - probably easier to remove in an atomic PR.